### DISCUSSION TITLE
Not able to deploy terrakube using docker-compose

### DISCUSSION THREAD

**Discussion by abhinavsquadstack** [terrakube-io/terrakube#463] [Q&A] [answered]
Hey Team, 
I am trying to deploy terrakube on aws ec2 machine (ubuntu x86) using docker-compose. I followed the documentation and changed /etc/hosts file as "EC2PublicIP terrakube-*". Although terrakube-ui landing page with login button was loading but whenever I am clicking on Login button, I am getting below error.
<img width="495" alt="image" src="https://github.com/AzBuilder/terrakube/assets/121935932/66627378-34fe-4cf9-9c7d-77254d07755f">

I need help here. Do I need to change some configuration?
Thanks

**Comment by alfespa17** [comment, marked as answer]
Not sure if I understand correctly what you are doing, you launch docker compose in a VM in AWS and your are trying to connect to terrakube from outside of the VM?

If that is the case I guess you need to edit the /etc/host inside the VM and the /etc/host in the computer that your are using TO connect to terrakube

**Reply by abhinavsquadstack** [reply]
Thanks for reply
